css/style.css

```diff
@@ -22,11 +22,12 @@ section.landing {
   margin:0;
   padding:0;
   background-size: cover;
-  background-color:rgba(0,46,102,.8);
+  //background-color:rgba(0,46,102,.8);
+  color:black;
 }
   
 
-img{
+img {
     max-width:120%;
```

Comment:
`max-width` should be 100%. Never 120%. Unless there's a reason you chose to do this?